app/apphandlers/embeddedAppHandler.go

```diff
@@ -59,7 +59,7 @@ func (embedded *EmbeddedApp) StartRudderCore(ctx context.Context, options *app.O
 
 	var gatewayDB jobsdb.HandleT
 	var routerDB jobsdb.MultiTenantHandleT
-	var batchRouterDB jobsdb.MultiTenantHandleT
+	var batchRouterDB jobsdb.HandleT
```

Comment:
Why we are reverting back this change?

Reply:
Changed it since we have done no changes in batch router queries..